src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -41,8 +41,10 @@ public class BatchHandler : DatabaseRequestHandler
         [RavenAction("/databases/*/bulk_docs", "POST", AuthorizationStatus.ValidUser)]
         public async Task BulkDocs()
         {
+            bool forceRevisionCreation = GetBoolValueQueryString("forceRevisionCreation", required: false) ?? false;
```

Comment:
This works well from Studio, but it will be problematic from Client API, because it is not per item, but per whole batch.
It needs to be either a new command or put command with a flag.

Reply:
Changed this flag to be part of the commandData item